src/main/java/seedu/duke/model/ItemContainer.java

```diff
@@ -139,7 +139,8 @@ public void updateItem(Item originalItem, Item updatedItem)
      *
      * @param name The specified name of Item
      * @return Item with the specified name
-     * @throws ItemNotExistException If the Item does not exist in the ItemContainer
+     * @throws NullPointerException if the input name is null
```

Comment:
No need to specify NullPointerException i think

Reply:
deleted